src/data/faq_de.json

```diff
@@ -1342,6 +1342,35 @@
                 "title": "Anwendung",
                 "id": "application",
                 "accordion": [
+                    {
+                        "title": "Fragen zu Quarantäne-Maßnahmen",
+                        "anchor": "quarantine_measures",
+                        "active": true,
+                        "textblock": [
+                            "Sollten Sie positiv getestet werden, muss Ihr zuständiges Gesundheitsamt informiert werden. Dies geschieht über Ihren Hausarzt bzw. das Testzentrum, bei dem Sie Ihren Test durchgeführt haben. Wurde Ihr Test nicht in Deutschland durchgeführt, sind Sie verpflichtet, Ihr zuständiges Gesundheitsamt oder Ihren Hausarzt selbst zu informieren, wenn das Testergebnis positiv sein sollte. Die zuständige Behörde, in der Regel Ihr Gesundheitsamt, wird dann eine Isolation anordnen.",
+                            "Die Quarantäne, bzw. Isolation im Falle eines positiven Testergebnisses, ist ein wichtiges Element in der Bekämpfung der Pandemie und soll die Verbreitung des Virus verhindern. Sie dient Ihrem Schutz und dem Schutz Aller vor Ansteckung mit dem neuartigen Coronavirus.",

```

Comment:
Ist das Virus nach 2 Jahren  immer noch neuartig?

Reply:
Time is relative.